client/packages/invoices/src/Prescriptions/api/hooks/utils.test.ts

```diff
@@ -896,66 +896,4 @@ describe('Allocated quantities - assign prescribed quantity', () => {
     expect(lineWithPrescribedQuantity?.length).toBe(1);
     expect(lineWithPrescribedQuantity?.[0]?.prescribedQuantity).toEqual(15);
   });
-
-  it('should assign prescribedQuantity to the next line if current line gets deleted', () => {
```

Comment:
test is dropped as well, as we are really hacking the test to think that a line has gotten deleted if `numberOfPacks` is zero. Now we support the case of saving an item if `numberOfPacks` is zero